helm/ako/templates/statefulset.yaml

```diff
@@ -42,6 +42,12 @@ spec:
         persistentVolumeClaim:
           claimName: {{ .Values.persistentVolumeClaim }}
       {{ end }}
+      {{ if gt (len .Values.image.pullSecrets) 0 }}
+      {{- with .Values.image.pullSecrets }}
```

Comment:
Is length check enough here?

Reply:
@swathinsankaran This condition also works if pullSecrets field is not specified at all. Do we need anything else?

Reply:
I was under the impression that `len(param) > 0` would be enough.

Reply:
I have removed both the if condition and the with block. None of those were actually required.